### DISCUSSION TITLE
Tool Tips for Page Buttons

### DISCUSSION THREAD

**Discussion by deefdragon** [linuxserver/Heimdall#688] [Ideas]
It is very difficult to determine what the different icons in the bottom corner are for new users. Even with using it for a bit, I still get confused as to what does what, and basically have to go down the line to find what I am looking for.

Please add tool-tips to the buttons, or Pop out the button titles (or both).

Memorizing what the icons do would be an ideal end goal for a user, but adding tool-tips will improve new user experience.

**Comment by KodeStar** [comment]
Agreed

**Comment by hackitz** [comment]
same would like this option or feature

**Comment by KodeStar** [comment]
This will be included in 2.5.0
